src/opendp/test/regression_tests.tcl

```diff
@@ -47,4 +47,5 @@ record_tests {
   aes
   gcd
   ibex
+  spm
```

Comment:
spm doesn't convey the intent of the test.
I would much rather see a smaller more targeted test named multi_height05 to be consistent with the other tests.